### DISCUSSION TITLE
Placeholder colour for floating labels

### DISCUSSION THREAD

**Discussion by lonix1** [twbs/bootstrap#39394] [Ideas] [answered]
For floating labels: in the non-floating position it is dark, and in the floating position it is light.

But a placeholder is typically light to show that it's not real text.

Is it possible to set it to light in the non-floating position too?

**Comment by julien-deramond** [comment]
Are you looking for a technical solution to do it on your website using Bootstrap, or is it a suggestion to improve the default design side of Bootstrap?

**Reply by lonix1** [reply]
Both :) I'm not using SCSS so I didn't find a way to do it without ugly hacks.

I was surprised that in the non-floating state it is a dark colour. I thought it must be a mistake on my side - but when I checked the docs I found that it is the same (dark).

**Comment by lonix1** [comment]
Am I allowed to "convert" this into an issue or is that something only repo owners can do? In which case I hope it's ok if I create a new issue because I suspect this is a bug.

**Reply by julien-deramond** [reply]
After a quick search, I haven't found any explicit issues in our backlog asking to change this color (only https://github.com/twbs/bootstrap/pull/39172 is kind of linked to that). You can create a new issue for that and I'll label it "design" and we'll see what is the answer of our designer.

**Reply by lonix1** [reply, marked as answer]
Converted here: https://github.com/twbs/bootstrap/issues/39420
